server/src/main/java/com/tencent/rss/server/ShuffleServerConf.java

```diff
@@ -42,9 +42,9 @@
       .noDefaultValue()
       .withDescription("Spill threshold for buffer manager, it must be less than rss.server.buffer.capacity");
 
-  public static final ConfigOption<Integer> SERVER_PARTITION_BUFFER_SIZE = ConfigOptions
+  public static final ConfigOption<Long> SERVER_PARTITION_BUFFER_SIZE = ConfigOptions
       .key("rss.server.partition.buffer.size")
-      .intType()
+      .longType()
```

Comment:
like SERVER_READ_BUFFER_CAPACITY, many configs's default value is long. Should we replace these with kb.mb.gb.

Reply:
This patch only addresses size in bytes, in which long is required.

Reply:
Fixed in config file